www/resources/views/pages/testhistoryadmin.blade.php

```diff
@@ -89,7 +89,7 @@
                             </b>
                         @endif
                     </td>
-                    <td class="location">{!! $location !!}
+                    <td class="location scrollable-td"><div class="scrollable-td-content">{!! $location !!}</div>
```

Comment:
@v-hclaveria is this expected that the below span is outside the div?

Reply:
please also ensure that those changes are not impacting table on other browsers

Reply:
![image](https://github.com/catchpoint/WebPageTest/assets/149002067/e4451984-2ab0-433c-9e45-92ffe5c71ae3)